softwareComponents/torqueComputation/src/compute.cpp

```diff
@@ -0,0 +1,465 @@
+#include <torqueComputation/compute.hpp>
```

Comment:
Compute je strašně obecný název pro soubor, mělo by se to jmenovat podle toho co to počítá